docs/blog/2022-08-12-observing-randomness.md

```diff
@@ -0,0 +1,140 @@
+
+As a little bit of background, a drand *network* is composed of multiple *nodes*. Each network generates one (or more) *beacons*, each of which generates separate randomness beacons at a specific interval, called a *round*. The number of nodes associated with each beacon is fixed at any given point in time and is only changed during a *ceremony*. During the ceremony a *threshold* is set, which is the minimum number of nodes that need to be connected to the network at any given time for randomness generation to succeed.
+
+## The need for monitoring
+
+To ensure security, we do not allow just any host to join the network at any time. Whenever a host joins or leaves the network, we hold what is called a “ceremony”, where cryptographic keys are refreshed and exchanged. In between ceremonies the list of hosts that participate in a drand network is static. Ceremonies are usually held quarterly, regardless of whether any node joins or leaves the network. Before each ceremony, an allowlist is distributed to all the participants, which contains the IP addresses of all nodes in the network. This is done so that LoE members can open the appropriate firewall ports.
+
+One issue with the ceremonies is that it is a synchronous operation where all node operators need to be at the keyboard at the same (window of) time and execute certain commands in their nodes at the correct moment and in the correct order. Since node operators are based around the world, it is a challenge to coordinate a time that is convenient for everyone, and that does not even account for unforeseen reasons of why a node operator might be unavailable. In the past this has led to delaying or even postponing some ceremonies.
+
+There are other things that can go wrong during a ceremony. One of the nodes might not be running the appropriate version of `drand`. The node operator might restart their node in the middle of the ceremony, which then puts their node in a wrong state to participate in a ceremony. There might be connectivity problems which do not allow a node to correctly participate. And so on and so forth. Up until now, the coordination between operators has been through off-band channels (such as email or Slack), which resulted in postponed or cancelled ceremonies. The difficulties in coordinating a ceremony through such means surfaced the need to have full visibility into the drand network and the state of each node participating in it, in order to ensure that all nodes are in the correct state and that everything happens at the right time and in the right order.
```

Comment:
The reasons might be clearer in a bullet point list

Reply:
I thought of putting it in text since it's not an exhaustive list, I think that it reads better this way.